src/buildChecks/constants.js

```diff
@@ -1,3 +1,13 @@
 const { createObject } = require('../utils');
 
-module.exports = createObject(['BUILD_TIME', 'APP_SIZE']);
+module.exports.buildMetrics = createObject(['BUILD_TIME', 'APP_SIZE']);
```

Comment:
define a way to inject these variables somehow

Reply:
Yup, we are going to define two new args to define the buildPath and the extraBuildParams. @alejobh Is going to apply it in the next PR